### PR TITLE
[regression-test](p0) add p0 test cases

### PR DESCRIPTION
**Pull request by LizZhang315**
# Proposed changes

Add p0 test cases, including:
aggregate
join
union
order by
group by
keyword
arithmetic operators
logical operators
case function
coalesce
between
in
like
limit
where
regexp
window function
runtime filter
schema change

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [ ] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [ ] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

This is only part of the regression test cases.

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: regression-test/plugins/plugin_example.groovy [hunk 1]
```diff
@@ -32,3 +32,23 @@ Suite.metaClass.testPlugin = { String info /* param */ ->
 }
 
 logger.info("Added 'testPlugin' function to Suite")
+
+Suite.metaClass.check2_palo = { Object res1, Object res2 /* param */ ->
```
Comment (zhangy5) [inline regression-test/plugins/plugin_example.groovy:36]: avoid using palo, use doris instead

**Comment by yiguolei** [comment]
wow, its great

**Comment by hello-stephen** [comment]
great job~

**Review by morningman — APPROVED** [review]
LGTM

**Comment by morningman** [comment]
#11787 
#11786
There are 2 issues need to be fixed.

**Pull request merged**